src/placeos-compiler/git.cr

```diff
@@ -56,7 +56,14 @@ module PlaceOS::Compiler
     end
 
     def self.commits(file_name : String | Array(String), repository : String, working_directory : String, count : Int32 = 50) : Array(Commit)
-      base_arguments = ["log", "--format=format:%h%n%cI%n%an%n%s%n<--%n%n-->", "--no-color", "-n", count.to_s, "--"]
+      base_arguments = [
+        "log",
+        "--format=format:%h%n%cI%n%an%n%s%n<--%n%n-->",
+        "--no-color",
+        "-n", count.to_s,
+        "origin",
```

Comment:
This will end up mapping to `refs/remotes/origin`, which then points to the default branch rather than the branch the repo is using. Will need to bubble the actual branch name through to here (I think). May be worth a quick parse of https://www.git-scm.com/docs/gitrevisions to confirm.

Reply:
I don't see any reference to that defaulting to the default branch

Reply:
Ideally this would just get the origin head of the current ref, can't see a nice way to do that without another command here